data/json/items/tools.json

```
@@ -4002,7 +4002,8 @@
           "light_plus_battery_cell",
           "light_minus_battery_cell",
           "light_atomic_battery_cell",
-          "light_minus_atomic_battery_cell"
+          "light_minus_atomic_battery_cell",
+          "light_disposable_cell"
```

Nice catch!  Since `light_minus` cells are valid, how about adding that disposable version too?

```suggestion
          "light_disposable_cell",
          "light_minus_disposable_cell"
```

---

I actually didn't know they had a disposable form, don't think I've seen the tiny ones before.  Hopefully I do this right, just click commit suggestion?  Fingers crossed.